### PR TITLE
a small refactoring flutter log view

### PR DESCRIPTION
**Pull request by quangson91**
I make some small refactoring flutter log view

// cc @pq for review.

### CODE REVIEW

**Comment by pq** [comment]
🎆 Thanks!  On holiday here but will look tomorrow or Monday at the latest.

**Comment by quangson91** [comment]
Thank you!

File: src/io/flutter/logging/FlutterLogView.java [hunk 1]
```diff
@@ -120,44 +121,34 @@ public ConfigureAction() {
...
-      );
-      presentation.putClientProperty("button", actionButton);
-      return actionButton;
+    @Override
+    public void actionPerformed(@NotNull AnActionEvent e) {
+      getComponentOfActionEvent(e).ifPresent(component -> {
+        final ActionPopupMenu popupMenu = ActionManager.getInstance().createActionPopupMenu(ActionPlaces.UNKNOWN, actionGroup);
+        popupMenu.getComponent().show(component, component.getWidth(), 0);
+      });
     }
```
Comment (pq) [inline src/io/flutter/logging/FlutterLogView.java]: Oh wow.  This is never used.  Thanks!

File: src/io/flutter/logging/FlutterLogView.java [hunk 2]
```diff
@@ -464,19 +455,16 @@ private Object getTagForPosition(MouseEvent e) {
...
-    logTree.getColumn(SEQUENCE).setMinWidth(50);
-    logTree.getColumn(SEQUENCE).setMaxWidth(50);
-    logTree.getColumn(LEVEL).setMinWidth(70);
-    logTree.getColumn(LEVEL).setMaxWidth(70);
-    logTree.getColumn(CATEGORY).setMinWidth(110);
-    logTree.getColumn(CATEGORY).setMaxWidth(110);
+    fixColumnWidth(logTree.getColumn(TIME), 100);
+    fixColumnWidth(logTree.getColumn(SEQUENCE), 50);
+    fixColumnWidth(logTree.getColumn(LEVEL), 70);
+    fixColumnWidth(logTree.getColumn(CATEGORY), 110);
```
Comment (pq) [inline src/io/flutter/logging/FlutterLogView.java]: 💎

File: src/io/flutter/logging/FlutterLogView.java [hunk 3]
```diff
@@ -364,25 +355,27 @@ public void updated(int filtered, int total) {
       }
     }
 
-
-    String countString(int count) {
+    @NotNull
+    private String countString(int count) {
```
Comment (pq) [inline src/io/flutter/logging/FlutterLogView.java]: Aside: this could be `static` too...

**Pull request merged**